cpp/test/CMakeLists.txt

```diff
@@ -197,14 +196,15 @@ endif()
 
 if(BUILD_CUML_MG_TESTS)
 
-  ConfigureTest(PREFIX MG NAME KMEANS_TEST PATH mg/kmeans_test.cu OPTIONAL NCCL CUMLPRIMS ML_INCLUDE)
+  # This test needs to be rewritten to use the MPI comms, not the std comms.
```

Comment:
I think it's fine to comment this out, but can you also create a github issue just so it doesn't get forgotten and reference the issue # here?

Reply:
Added link to GH issue